histomicstk/features/ComputeMorphometryFeatures.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+import pandas as pd
+from skimage.measure import regionprops
+
+
+def ComputeMorphometryFeatures(im_label):
+    """
+    Calculates morphometry features for each object
+
+    Parameters
+    ----------
+    im_label : array_like
+        A labeled mask image wherein intensity of a pixel is the ID of the
+        object it belongs to. Non-zero values are considered to be foreground
+        objects.
+
+    rprops[i] : property
+        A rprops[i] property from regioprops.
+
+    Returns
+    -------
+    fdata: Pandas data frame containing the following morphometry features
+    for each object/label
+        'Area',
+        'Circularity',
+        'Eccentricity',
+        'EquivalentDiameter'
+        'Extent',
+        'MajorAxisLength',
+        'MinorAxisLength',
+        'MajorMinorAxisRatio',
+        'Perimeter',
+        'Solidity',
+    """
```

Comment:
@cooperlab @slee172 Lee and sanghoon Can you please take a look? I refactored morphometry features.

I removed CentroidsX, CentroidsY as these are absolution locations and are not invariant to spatial transforms. [Local centroid](https://github.com/scikit-image/scikit-image/blob/master/skimage/measure/_regionprops.py#L420-L422) might be informative but even that would not be properly normalized unless it is computed relative to some oriented bounding box.

I add [EquivalentDiameter](https://github.com/scikit-image/scikit-image/blob/master/skimage/measure/_regionprops.py#L397-L398)